### PR TITLE
[WIP] component readme updates

### PR DESCRIPTION
**Pull request by nstsj**
- updated I/O on several components + made templates for all the components that had no readmes. 
- gonna add I/O info  and dependencies info in the upcoming pushes

### CODE REVIEW

File: annotators/COMeT/README.md [hunk 1]
```diff
@@ -5,6 +5,10 @@
 COMeT is a Commonsense Transformers for Automatic Knowledge Graph Construction service based
 on [comet-commonsense](https://github.com/atcbosselut/comet-commonsense) framework written in Python 3.
 
+<<<<<<< HEAD
+
+=======
+>>>>>>> dev
```
Comment (dilyararimovna) [inline annotators/COMeT/README.md:11]: некорректное подмерживание изменений!

**Review by dilyararimovna — CHANGES_REQUESTED** [review]
- проверить подмерживание дева - некорректные файлы есть
- почему некотоыре файлы не заполнены? они будут заполнены или нет?

File: annotators/COMeT/README.md (hunk 1 above)
Comment (dilyararimovna) [reply annotators/COMeT/README.md:14]: @nstsj

**Comment by dilyararimovna** [comment]
@nstsj merge dev and resolve conflicts.
BTW previously you merged dev incorrectly, because in Files of this PR you can see a lot of files that you did not touch.

File: skill_selectors/post_annotator_selector/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+To avoid resource overload, the number of selected skills is controlled.
+
+
+## Input/Output
+
+**Input**
+Annotated user input and dialogue context
+
+**Output**
+A list of selected skills
```
Comment (dilyararimovna) [inline skill_selectors/post_annotator_selector/README.md:19]: ну если присмотреться к тому, что он выбирает в пайплайне, можно увидеть, ЧТО НЕ СКИЛЛЫ

File: response_selectors/ranking_based_response_selector/README.md [hunk 3]
```diff
@@ -15,4 +16,29 @@ Parameter `FILTER_TOXIC_OR_BADLISTED` defines whether it filers out toxic hypoth
 
 ## Dependencies
 
-- either candidate annotations by `SENTENCE_RANKER_ANNOTATION_NAME` or service `SENTENCE_RANKER_SERVICE_URL`.
\ No newline at end of file
+- either candidate annotations by `SENTENCE_RANKER_ANNOTATION_NAME` or service `SENTENCE_RANKER_SERVICE_URL`.
+<<<<<<< HEAD
```
Comment (dilyararimovna) [inline response_selectors/ranking_based_response_selector/README.md:20]: ОПЯТЬ

File: annotators/COMeT/README.md (hunk 1 above)
Comment (dilyararimovna) [reply annotators/COMeT/README.md:14]: это по-прежнему тут

**Review by dilyararimovna — CHANGES_REQUESTED** [review]

File: annotators/BadlistedWordsDetector/README.md [hunk 4]
```diff
@@ -4,8 +4,8 @@
 Spacy-based user utterance annotator that detects words and phrases from the badlist
 
 ## I/O
-input:  "sentences": ["fucking hell", "he mishit the shot", "you asshole"],
-output: words and their tags
+**Input:**  a list of user's utterances
 [{"bad_words": True}, {"bad_words": False}, {"bad_words": True}]
```
Comment (dilyararimovna) [inline annotators/BadlistedWordsDetector/README.md:8]: where is the title output?

File: annotators/COMeT/README.md [hunk 5]
```diff
@@ -36,6 +37,22 @@ docker-compose -f docker-compose.yml -f local.yml exec comet-conceptnet bash tes
...
+- currentUtterance: latest reply from a user
+- pastResponses: a list of user's utterances 
+
+**Output**
+a list of probabilities about the utterance:
+- isResponseComprehensible
+- isResponseErroneous
+- isResponseInteresting
+- isResponseOnTopic
+- responseEngagesUser
```
Comment (dilyararimovna) [inline annotators/COMeT/README.md:53]: that;'s not about comet

File: annotators/DeepPavlovEmotionClassification/README.md [hunk 6]
```diff
@@ -1 +1,15 @@
...
+
+## Description
+
+BERT Base model for emotion classification which learned at the custom dataset (described more precisely in (our article)[link here])
+
+## I/O
+
+**Inpunt**
+
+**Output:**
```
Comment (dilyararimovna) [inline annotators/DeepPavlovEmotionClassification/README.md:11]: ?

File: annotators/DeepPavlovFactoidClassification/README.md [hunk 7]
```diff
@@ -0,0 +1,9 @@
+# Title
+## Description
+
+## Input/Output
+
+**Input**
+**Output**
```
Comment (dilyararimovna) [inline annotators/DeepPavlovFactoidClassification/README.md:7]: ?

File: annotators/NER/README.md [hunk 8]
```diff
@@ -0,0 +1,9 @@
+# Title
+## Description
+
+## Input/Output
+
+**Input**
+**Output**
```
Comment (dilyararimovna) [inline annotators/NER/README.md:7]: ?

File: annotators/property_extraction/README.md [hunk 9]
```diff
@@ -21,9 +21,19 @@ import requests
...
+```
 
->>> [
+**Output example**
+```
+[
     {"triplets": [{"subject": "user", "relation": "like activity", "object": "walking"}, {"subject": "user", "relation": "have pet", "object": "two dogs"}]}, 
     {"triplets": [{"subject": "user", "property": "marital status", "object": "husband"}, {"subject": "user", "relation": "like activity", "object": "travel"}]}
     ]
 ```
```
Comment (dilyararimovna) [inline annotators/property_extraction/README.md:32]: why somewhere examples and somewhere textual description? (like custom EL)

File: response_selectors/convers_evaluation_based_selector/README.md [hunk 10]
```diff
@@ -36,6 +38,20 @@ LANGUAGE: language to consider
 FALLBACK_FILE: a file name with fallbacks from `dream/common/fallbacks/`
 ```
 
-## Dependencies
+## Input/Output
+**Input:** a list of hypotheses with their scores and metadata
+**Output:** a list of selected hypotheses with their scores and metadata
```
Comment (dilyararimovna) [inline response_selectors/convers_evaluation_based_selector/README.md:43]: not a list but the final hypothesis actually

File: response_selectors/convers_evaluation_based_selector/README.md [hunk 11]
```diff
@@ -36,6 +38,20 @@ LANGUAGE: language to consider
...
+Then use `--url`.
 
+Example of usage with url:
+
+## Dependencies
+```
+python response_selectors/convers_evaluation_based_selector/measure_quality.py \
+                   --data_dir response_selectors/convers_evaluation_based_selector/labeled_data/  \
+                   --url http://0.0.0.0:8009/respond
+```
```
Comment (dilyararimovna) [inline response_selectors/convers_evaluation_based_selector/README.md:57]: remove all this info. It is not about local usage.

File: response_selectors/ranking_based_response_selector/README.md [hunk 12]
```diff
@@ -15,4 +15,29 @@ Parameter `FILTER_TOXIC_OR_BADLISTED` defines whether it filers out toxic hypoth
...
+**Input:**
+text here
+
+**Output:** Ranking_based_response_selector service returns
+- the selected skill’s name,
+- the response text (which can be overwritten)
+- the confidence level
+- the selected skill name,
+- the response text (which can be overwritten)
+- the confidence level
```
Comment (dilyararimovna) [inline response_selectors/ranking_based_response_selector/README.md:36]: better to put an example

File: response_selectors/ranking_based_response_selector/README.md [hunk 13]
```diff
@@ -15,4 +15,29 @@ Parameter `FILTER_TOXIC_OR_BADLISTED` defines whether it filers out toxic hypoth
...
+=======
+# Ranking_based_response_selector
+
+## Description
+
+
+## Input/Output
+
+**Input:**
+text here
```
Comment (dilyararimovna) [inline response_selectors/ranking_based_response_selector/README.md:28]: ?

**Review by dilyararimovna — CHANGES_REQUESTED** [review]
A lot of empty files. fill in. And fix already filled in

File: response_selectors/ranking_based_response_selector/README.md (hunk 12 above)
Comment (nstsj) [reply response_selectors/ranking_based_response_selector/README.md:36]: an example added

File: response_selectors/ranking_based_response_selector/README.md (hunk 13 above)
Comment (nstsj) [reply response_selectors/ranking_based_response_selector/README.md:28]: wrong file upd after rebasing, fixed

File: response_selectors/convers_evaluation_based_selector/README.md (hunk 11 above)
Comment (nstsj) [reply response_selectors/convers_evaluation_based_selector/README.md:57]: i need clarification here: do we want to remove the whole paragraph on "### How to run conversation evaluator locally" or just lines 54-57?

File: response_selectors/convers_evaluation_based_selector/README.md (hunk 10 above)
Comment (nstsj) [reply response_selectors/convers_evaluation_based_selector/README.md:43]: Fixed to a correct updated version.

File: annotators/NER/README.md (hunk 8 above)
Comment (nstsj) [reply annotators/NER/README.md:7]: some readmes respawn in their old versions after the branch rebasing(this is a perpetual issue). 
Fixed this one as well.

File: annotators/DeepPavlovFactoidClassification/README.md (hunk 7 above)
Comment (nstsj) [reply annotators/DeepPavlovFactoidClassification/README.md:7]: For this one need to ask someone if/how this one is different from the Factoid clf in `combined classification` (the info on this one is scarce in our repo). 
Pinged in our slack channel.

File: annotators/DeepPavlovEmotionClassification/README.md (hunk 6 above)
Comment (nstsj) [reply annotators/DeepPavlovEmotionClassification/README.md:11]: same as Factoid: the info on this one is scarce in our repo, made a ping in our slack channel

File: annotators/COMeT/README.md (hunk 5 above)
Comment (nstsj) [reply annotators/COMeT/README.md:53]: old-old-thousands-years-old version of a readme respawned after rebasing. 
Fixed to a correct, current one.

Comment (nstsj) [reply annotators/COMeT/README.md:53]: Fixed!

File: annotators/BadlistedWordsDetector/README.md (hunk 4 above)
Comment (nstsj) [reply annotators/BadlistedWordsDetector/README.md:8]: fixed

File: annotators/custom_entity_linking/README.md [hunk 14]
```diff
@@ -4,29 +4,40 @@
...
+processed information about:
 - entities
 - entity_id (ids for multiple entities)
 - entity_confidence score
 - entity_id_tags
-  
+
+An output example:
+```
+```
```
Comment (dilyararimovna) [inline annotators/custom_entity_linking/README.md:38]: missed

File: skills/dummy_skill/README.md [hunk 15]
```diff
@@ -2,14 +2,21 @@
...
+```
+
+```
 **Output**
 
-## Dependencies
 
+```
+
+```
```
Comment (dilyararimovna) [inline skills/dummy_skill/README.md:18]: missed

File: skills/dummy_skill/README.md [hunk 16]
```diff
@@ -2,14 +2,21 @@
...
-Russian Random questions are collected from https://mensby.com/women/relations/150-voprosov-chtoby-luchshe-uznat-sobesednika-ili-sobesednicu
-and https://habr.com/ru/company/testutor/blog/298180/
+TBA
 
 # I/O
 **Input**
+A user's utterance
+```
+
+```
```
Comment (dilyararimovna) [inline skills/dummy_skill/README.md:12]: missed

File: skills/personal_info_skill/README.md [hunk 17]
```diff
@@ -0,0 +1,20 @@
...
+A user utterance
+```
+["my name is john."]
+```
+**Output**
+A parsed utterance, retrieved name and a reply
+
+```
+ [["Nice to meet you, John."],
+ ["text": "john", "type": "PER"]]
```
Comment (dilyararimovna) [inline skills/personal_info_skill/README.md:16]: no. it accepts annotated utterances, including NER.

**Review by dilyararimovna — CHANGES_REQUESTED** [review]

**Review by dilyararimovna — APPROVED** [review]
Not everything is fixed but let's merge it because it took so long of preparing pr

**Pull request merged**